instrumentation/rmi/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rmi/context/ContextPayload.java

```diff
@@ -37,7 +34,9 @@ public ContextPayload(Map<String, String> context) {
 
   public static ContextPayload from(Context context) {
     ContextPayload payload = new ContextPayload();
-    tracer().inject(context, payload, SETTER);
+    GlobalOpenTelemetry.getPropagators()
+        .getTextMapPropagator()
+        .inject(context, payload, ContextPayloadSetter.INSTANCE);
```

Comment:
I think we should add `inject` method to `Instrumenter`. My problem with approach like this is that it is very easy to end up using different instances of `OpenTelemetry` .

Reply:
Can you create an issue for this?

Reply:
Done.